setup-nexus-npm/action.yml

```diff
@@ -0,0 +1,17 @@
+name: setup-nexus-npm
+inputs:
+  endpoint:
+    required: true
+  login:
+    required: true
+  password:
+    required: true
+runs:
+  using: composite
+  steps:
+    - run: |
+        rm -f ~/.npmrc
+        echo registry=https://${{ inputs.endpoint }}/repository/npm-txm >> ~/.npmrc
```

Comment:
Same comment as on the other PR: Can we use a heredoc here? https://mywiki.wooledge.org/HereDocument